contrib/spring/src/main/java/io/opencensus/contrib/spring/autoconfig/OpenCensusProperties.java

```diff
@@ -35,4 +36,47 @@ public boolean isEnabled() {
   public void setEnabled(boolean enabled) {
     this.enabled = enabled;
   }
+
+  public Trace getTrace() {
+    return this.trace;
+  }
+
+  public void setTrace(Trace trace) {
+    this.trace = trace;
+  }
+
+  /** Trace properties. */
+  public static class Trace {
```

Comment:
Does this class need to be mutable or would an appropriate constructor/builder be sufficient?

Reply:
it doesn't need to be mutable. I have made it final. Typically it would used by spring framework. Application code would not use it directly.